server/plugin/command_test.go

```diff
@@ -83,3 +83,72 @@ func TestValidateFeatures(t *testing.T) {
 		})
 	}
 }
+
+func TestParseCommand(t *testing.T) {
+	type output struct {
+		command    string
+		action     string
+		parameters []string
+	}
+
+	tt := []struct {
+		name  string
+		input string
+		want  output
```

Comment:
Nit pick: I'm not sure if an additional depth makes the code more readable. Should this be `expectedCommand`, `expectedAction`, `expectedParameters` instead?

Reply:
@hanzei This makes the comparison easier in the actual test logic. We can just compare the received and expected `output` structs as opposed to comparing each value one by one.

Reply:
Good point :+1: